en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+:::
+
+This is a screenshot for a multicotor:
+
+![Actuators MC (QGC)](../../assets/config/qgc_actuators_mc_aux.png)
+
+
+:::note
+If you find a setting missing (e.g. reversable motors), make sure to enable the **advanced** checkbox in the top right corner (not visible above).
+:::
+
+## Geometry
+The airframe pre-selects the geometry type, which can provide additional configuration options, such as the motors positions on a multicotor, or the control surface types on a plane.
+The UI reflects that and displays a customized view for each type.
+
+
+### Conventions
+The following sections contain the conventions and explanations for configuring the geometry.
+
+#### Coordinate system
+The coordinate system is NED, where the X axis points forward, the Y axis to the right and the Z axis down.
+Positions are relative to the center of gravity (in meters).
+
+#### Control Surfaces and Servo Direction
+Control surfaces use the following deflection direction convention:
+- horizontal (e.g. Aileron): up = positive deflection
+- vertical (e.g. Rudder): right = positive deflection
+- mixed (e.g. V-Tail): up = positive deflection
+
+![Plane Deflections](../../assets/config/plane_servo_convention.png)
+
+:::note
+If a servo does not move into the expected direction, reverse it via checkbox on the Actuator Output.
+:::
+
+
+### Tilt Servos
+Tilt servos are configured as following:
+- The reference direction is upwards (negative Z direction).
+- Tilt direction: **Towards Front** means the servo tilts towards positive X direction, whereas **Towards Right** means towards positive Y direction.
+- Minimum and maximum tilt angles: specify the physical limits of the tilt at minimum control and maximum respectively. An angle of 0 means to point upwards, then increases towards the tilt direction.
+  :::note
+  Negative angles are possible. For example tiltable multirotors have symmetrical limits and one could specify -30 as minimum and 30 degrees as maximum.
+  :::
+  :::note
+  If a motor/tilt points downwards and tilts towards the back it is logically equivalent to a motor pointing upwards and tilting towards the front.
+  :::
+- Control: depending on the airframe, tilt servos can be used to control torque on one or more axis (it's possible to only use a subset of the available tilts for a certain torque control):
+  - Yaw: the tilts are used to control yaw (generally desired).
+    If four or more motors are used, the motors can be used instead.
+  - Pitch: typically differential motor thrust is used to control pitch, but some airframes require pitch to be controlled by the tilt servos. Bicopters are among those.
+- Tiltable motors are then assigned to one of the tilt servos.
+
+![Tilt Axis](../../assets/config/tilt_axis.png)
+
+### Reversible Motors
+If reversible motor(s) are used (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half), make sure to select the **Reversible** checkbox for those motor(s).
```

Comment:
Might be worth showing a screenshot example since there is no reversible motor shown in the screenshots

Reply:
I added one

Reply:
How does this work? I mean elsewhere we talk about motors having a single direction and reversing is used to make the motor go in that correct direction.

Here it sounds like you're saying that it is part of the vehicle geometry that it must have a reversible motor/servo and this is how you define it. Is that right? I guess that makes sense for a ground vehicle in particular so it can go backwards.

This only appears when relevant for your selected frame?

Reply:
It's not a must, it's a feature. Yes for rovers it's most relevant, but there's various other use-cases, like a vtol pusher to go back and forwards.
It's generally an advanced option.

Reply:
Thanks. What I mean is, this will only be an option in geometry that is offered for frames where it makes sense - so you can't just decide that one motor in an MC is reversible because you haven't specified that in the actuator config file as an option. Right?

I'll update this section a bit.